src/ontology/cl.Makefile

```diff
@@ -279,7 +279,6 @@ imports/merged_import.owl: mirror/merged.owl imports/merged_terms_combined.txt
 	if [ $(IMP) = true ]; then $(ROBOT) merge -i $< \
```

Comment:
Do we still need the imports/merged_import.owl goal in cl.Makefile? is it not identical with the one in `Makefile`?

Reply:
good point - just checked, makefile removes CLO in addition to what is removed, which I imagine is a good thing, will remove this goal for cl.makefile